### PR TITLE
refactor: Make `deno_runtime::shared` module public

### PR DESCRIPTION
**Pull request by theswerd**
# Making shared public in deno_runtime

### CODE REVIEW

File: cli/lib.rs
Comment (bartlomieju) [inline cli/lib.rs:1]: This file needs to be reverted

**Review by bartlomieju — COMMENTED** [review]
This change was discussed in `#dev` channel on Discord

File: cli/lib.rs
Comment (theswerd) [reply cli/lib.rs:1]: ooops - done

**Review by bartlomieju — APPROVED** [review]
Thanks!

**Pull request merged**